pages/index.js

```diff
@@ -66,11 +66,19 @@ export default function Home({
               <p className="pt-4 font-bold pb-8">
                 September 22nd–25th • Miami Beach, Florida
               </p>
-              <Link href="https://assembly.urbit.org" passHref>
-                <a className="button-lg max-w-fit bg-green-400 text-white">
-                  Learn More
-                </a>
-              </Link>
+
+              <div class="flex flex-wrap">
+                <Link href="https://assembly.urbit.org" passHref>
+                  <a className="button-lg max-w-fit bg-wall-600 text-white mr-3 mb-8">
```

Comment:
Let's add `target="_blank"` to this. It sees urbit.org in the link so it isn't opening in a new tab, but it belongs outside the urbit.org family of sites IA.